x-pack/plugin/esql-core/src/main/java/org/elasticsearch/xpack/esql/core/index/IndexResolver.java

```diff
@@ -25,14 +25,8 @@
 import org.elasticsearch.index.mapper.TimeSeriesParams;
 import org.elasticsearch.transport.NoSuchRemoteClusterException;
 import org.elasticsearch.xpack.esql.core.QlIllegalArgumentException;
+import org.elasticsearch.xpack.esql.core.type.*;
```

Comment:
That's going to blow up I think.

Reply:
ah, crumb, I hate it when intellij does that.  and with the NOCOMMIT line still in here, precommit didn't get to the step that flags this.  Thanks for spotting it.